contrib/pyln-grpc-proto/pyln/grpc/node_pb2.py

```diff
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: node.proto
-# Protobuf Python Version: 4.25.1
```

Comment:
I think we have some problems with the versioning of the Python dependencies. However, you did not make any changes that should impact the grpc proto so you can safely remove them